EIPS/eip-2429.md

```diff
@@ -0,0 +1,830 @@
+---
+eip: 2429
+title: Secret Multisig Recovery
+author: Ricardo Guilherme Schmidt (ricardo3@status.im), Miguel Mota (@miguelmota), Vitalik Buterin (@vbuterin), naxe (@ngmachado)
+discussions-to: https://ethereum-magicians.org/t/social-recovery-using-address-book-merkle-proofs/3790/
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-12-07
+requires: 137, 191, 831, 1271, 1344, 2470
+---
+
+
+## Simple Summary
+
+Allows user to select a private list of ethereum addresses which can authorize a call protected by a secret only user knows. 
+Allows account contracts to set a private list of ethereum addresses that can authorize a call protected by a secret.
+
+## Abstract
+
+A bad experience of crypto-currencies in general is the lost or exposure of a private key, which can lead to irreversible situations.
```

Comment:
I guess _the lost_ should be replaced with _the loss_ in this sentence

Reply:
Thanks a lot for the review! There are probably other "English errors" in the text, specially in cohesion and correct derivation of words. I'm not a native English speaker and after the EIP is Final, not only a security audit is needed, but also an English audit :smile: 

I'll this open until a commit fixes it.